core/src/test/resources/applicationContext-dao-test.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
same comment about expanding the core package : here we are coupling the core to services provided by the persistence layer